### PR TITLE
Fix overlay proposal

### LINKED ISSUE #164: Add fps and graph overlay

### PR DESCRIPTION
**Pull request by soywiz**
Fixes #164

### CODE REVIEW

File: korge/src/commonMain/kotlin/com/soywiz/korge/view/FpsOverlay.kt [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        renderContext.debugLineRenderContext.line(
+            graphLeft, graphTop + overlayHeight.toFloat(),
+            graphLeft + overlayWidth.toFloat(), graphTop + overlayHeight.toFloat()
+        )
+
+        for (n in 0 until frequencies.size) {
+            val y = (overlayHeight - frequencies[n].toDouble().convertRange(
+                0.0, maxFps.toDouble(),
+                overlayHeightGap.toDouble(), overlayHeight.toDouble()
+            ).toFloat() + graphTop).toFloat()
```
Comment (soywiz) [inline korge/src/commonMain/kotlin/com/soywiz/korge/view/FpsOverlay.kt:68]: I think we should move graphTop to the start of the expression

Comment (soywiz) [reply korge/src/commonMain/kotlin/com/soywiz/korge/view/FpsOverlay.kt:68]: Like `graphTop + overlayHeight - ... expr` . And we should move expr to another variable I guess

Comment (RezMike) [reply korge/src/commonMain/kotlin/com/soywiz/korge/view/FpsOverlay.kt]: Just add this fix in another commit

File: korge/src/commonMain/kotlin/com/soywiz/korge/view/FpsOverlay.kt [hunk 2]
```diff
@@ -10,26 +10,22 @@ import com.soywiz.korim.color.*
 internal fun ViewsContainer.installFpsDebugOverlay() {
     val frequencies = Deque<Int>()
```
Comment (soywiz) [inline korge/src/commonMain/kotlin/com/soywiz/korge/view/FpsOverlay.kt:11]: KDS should expose `IntDeque` that doesn't do boxing

File: korge/src/commonMain/kotlin/com/soywiz/korge/view/FpsOverlay.kt [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+internal fun ViewsContainer.installFpsDebugOverlay() {
+    val frequencies = Deque<Int>()
+    var previousTime = DateTime.now()
+    var frames = 0
+    val MAX_FREQUENCIES_SIZE = 50
+    views.debugHandlers.add { ctx ->
+        val scale = ctx.ag.devicePixelRatio
+
+        val fontSize = 8.0 * scale
+        val currentTime = DateTime.now()
```
Comment (soywiz) [inline korge/src/commonMain/kotlin/com/soywiz/korge/view/FpsOverlay.kt:19]: We should use `PerformanceCounter` to have sub-millisecond precision

**Pull request merged**